### PR TITLE
Introduce mock component

### PR DESCRIPTION
**Pull request by mjakobczyk**
**Description**

Changes proposed in this pull request:

- introduce `mock` component for testing purposes

**Related issue(s)**

See: https://github.com/kyma-incubator/reconciler/issues/329

### CODE REVIEW

File: resources/mock/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# mock
+
+## Overview
+
+Mock component is being used for testing purposes.
```
Comment (mmitoraj) [inline resources/mock/README.md:5]:
```suggestion
The Mock component is used for testing purposes.
```

File: resources/mock/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# mock
```
Comment (mmitoraj) [inline resources/mock/README.md:1]:
```suggestion
# Mock
```

**Comment by mjakobczyk** [comment]
A new approach is going to be implemented so this new component will be unnecessary. Closing.

**Comment by mjakobczyk** [comment]
Reopening for a few tests still to be performed.

**Comment by mjakobczyk** [comment]
Tests have been performed. This PR is no longer needed. Closing.

**Pull request closed without merging**